azure-cpi.html.md.erb

```diff
@@ -115,7 +117,7 @@ Example of 10GB disk:
 ```yaml
 disk_pools:
 - name: default
-  disk_size: 10_240
+  disk_size: 10_240 # On Azure the disk size must be in the range [1 * 1024, 1023 * 1024].
```

Comment:
Above only mentions for the size of ephemeral_disk. And it does not mention that the size must be larger than 1024 MB. So we think we should add a comment here.